packages/react-magma-dom/src/components/TreeView/useTreeView.ts

```diff
@@ -63,8 +63,12 @@ export interface UseTreeViewProps {
   testId?: string;
   /**
    * Action that fires when an item is expanded or collapsed
+   * Return an Array list of itemIds of items that are expanded
```

Comment:
Just to match the description for onSelectedItemChange,
```suggestion
   * Return an array of itemIds of items that are expanded
   * Example: ['item0', 'item1', 'item3']
```